pkgs/development/python-modules/ntlm-auth/default.nix

```
@@ -28,7 +29,7 @@ buildPythonPackage rec {
 
   meta = with lib; {
     description = "Calculates NTLM Authentication codes";
-    homepage = https://github.com/jborean93/ntlm-auth;
+    homepage = "https://github.com/jborean93/ntlm-auth";
     license = licenses.lgpl3;
```

```suggestion
    license = licenses.mit;
```

From https://github.com/jborean93/ntlm-auth/releases/tag/v1.1.0

> License change to MIT